runtime/extensions/CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ if(ENABLE_LIGHTNING_KOKKOS)
         GIT_TAG        ${LIGHTNING_KOKKOS_GIT_TAG}
     )
 
+    set(PL_BACKEND "lightning_qubit")
```

Comment:
Why do we need this variable?

Reply:
This is a new variable used by Lightning to switch between backend devices. I'm also re-structuring the monorepo to enable compiling multiple backend devices.